src/rust/engine/src/scheduler.rs

```diff
@@ -44,6 +44,24 @@ type ObservedValueResult = Result<(Value, Option<LastObserved>), Failure>;
 // Root requests are limited to Select nodes, which produce (python) Values.
 type Root = Select;
 
+// When enabled, the interval at which all stragglers that have been running for longer than a
+// threshold should be logged. The threshold might become configurable, but this might not need
+// to be.
+const STRAGGLER_INTERVAL: Duration = Duration::from_secs(30);
```

Comment:
Maybe name this `STRAGGLER_LOGGING_INTERVAL` to clarify that it is the interval for logging of stragglers? Just `STRAGGLER_INTERVAL` could be confused with "stragger_threshold" since they both involve time.

Reply:
Will do.